xml/hwbp_registers.xml

```diff
@@ -245,7 +250,12 @@
             When set, every instruction completed or exception taken in M mode decrements \Fcount
             by 1.
         </field>
-        <field name="0" bits="8" access="R" reset="0" />
+        <field name="hit" bits="8" access="R/W" reset="0">
+            When this trigger matches, this optional bit becomes set. The
+            debugger can use it to see which trigger(s) matched, and is
+            responsible for clearing it. If the bit is not implemented, it
+            may still be writable, but won't become set by a match.
```

Comment:
still be writable by what? What is the point of making this bit writable if this behavior isn't implemented (when previously it was just `R`)?

Reply:
I'm imagining a trigger implementation in the memory sub-system rather than the CPU. The CPU then has a bunch of signals going to the trigger implementation. The trigger implementation may or may not implement the "hit" bit, and the CPU might not know whether it does or not. Maybe this is too far fetched and we should just say it's not writable when not implemented. I don't think it really matters to the debugger, so I went with being flexible.